packages/plugin-connector/src/build-plugins/preprocess.js

```diff
@@ -7,16 +7,18 @@ import autoImport from 'sveltekit-autoimport';
 export const evidencePlugins = () => {
 	const componentPlugins = getPluginComponents();
 
-	const packages = componentPlugins.then((components) => {
-		/** @type {Record<string,string[]>} */
-		const packages = {};
-		for (const [component, data] of Object.entries(components)) {
-			if (!packages[data.package]) packages[data.package] = [];
-			const import_name = data.aliasOf ? `${data.aliasOf} as ${component}` : component;
-			packages[data.package].push(import_name);
-		}
-		return packages;
-	});
+	const packages = componentPlugins
+		.then((components) => {
+			/** @type {Record<string,string[]>} */
+			const packages = {};
+			for (const [component, data] of Object.entries(components)) {
+				if (!packages[data.package]) packages[data.package] = [];
+				const import_name = data.aliasOf ? `${data.aliasOf} as ${component}` : component;
+				packages[data.package].push(import_name);
+			}
+			return packages;
+		})
+		.catch(() => ({}));
```

Comment:
What was the error coming from here? Is it safe to handle quietly?

Reply:
This is happening when there is no evidence.plugins.yaml, but it is only a problem in the monorepo; there are other places that catch this error when in actual use. We want to prevent the svelte language server from crashing when it is missing.